sympy/physics/vector/tests/test_dyadic.py

```diff
@@ -95,3 +95,10 @@ def test_dyadic_subs():
 
 def test_check_dyadic():
     raises(TypeError, lambda: _check_dyadic(0))
+
+
+def test_dyadic_evalf():
+    N = ReferenceFrame('N')
+    s = symbols('s')
+    a = 5* s *(N.x | N.x)
+    assert a.evalf() == 5.0 * s * (N.x | N.x)
```

Comment:
Please add tests showing that `vec.evalf(10, subs={s: 5.124})` works as expected for both dyadic and vector.

Reply:
Currently added EvalMixin class doesn't bring the subs feature to evalf due to a bug, pr #20321 is working to add subs feature to all classes inheriting EvalMixin.

Reply:
I think a new issue should be opened to track that subs is working for Vector and Dyadic and test cases can be added when pr#20321 is merged.

Reply:
Yes, thank you.

Reply:
I have opened new issue to track subs #20437